gse-demo/src/main/java/com/powsybl/gse/demo/GseDemo.java

```diff
@@ -9,18 +9,102 @@
 import com.powsybl.gse.app.GseApp;
 import org.slf4j.bridge.SLF4JBridgeHandler;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.io.PrintWriter;
+
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
 public final class GseDemo {
 
+    private static final String UNCAUGHT_EXCEPTION_MESSAGE = "FATAL: uncaught throwable in javafx startup";
+    private static final String STARTUP_ERROR_LOGFILE_NAME = "GSE_STARTUP_ERROR.log";
+
     private GseDemo() {
     }
 
+    //The system is utterly broken. Try as many things as we can to get some info to the user.
+    private static void lastResortFallbackErrorLog(Throwable t) {
+        //We don't even have a logging system, this happens for example
+        //if we are missing jars or if the magic javalauncher has a problem.
+        //Try to show the error everywhere so that the user has the best chance of noticing it...
+        try {
+            //Try stdout
+            System.out.println(UNCAUGHT_EXCEPTION_MESSAGE);
```

Comment:
Replace `System.out` by `System.err`

Reply:
I chose System.out because java's launcher writes to stdout, so everything is in the same place:
```
    //  GseDemo Error invoking method.
    //  GseDemo Failed to launch JVM
```

It doesn't matter much for us because this is a GUI, so our stdout is not used by other programs. Also the config of logback logs everything to stdout.

You still want stdout ?